platform-sdk/swirlds-common/src/main/java/com/swirlds/common/threading/pool/StandardWorkGroup.java

```diff
@@ -178,6 +182,10 @@ public void handleError(final Throwable ex) {
             if (!exceptionHandled) {
                 logger.error(EXCEPTION.getMarker(), "Work Group Exception [ groupName = {} ]", groupName, ex);
             }
+            // Log to stderr for testing purposes
+            if (logExceptionsToStdErr) {
+                ex.printStackTrace(System.err);
+            }
```

Comment:
I am not sure about exception handling, but this new block seems to be safe to move into `!exceptionHandled`

Reply:
Nevermind